lang/LangSource/PyrObject.cpp

```diff
@@ -110,7 +110,7 @@ PyrSymbol *s_identitydictionary;
 PyrSymbol *s_linkedlist;
 PyrSymbol *s_sortedlist;
 PyrSymbol *s_array;
-PyrSymbol *s_list, *s_method, *s_fundef, *s_frame, *s_class;
+PyrSymbol *s_list, *s_rest, *s_method, *s_fundef, *s_frame, *s_class;
```

Comment:
Fair enough... you've spent more time on language code than I have. I had just seen classes accessed by `s_` variables and I assumed that was "the" way to do it. No objection if someone changes it.